handout/modules/shared-di-tree.md

```diff
@@ -39,24 +39,8 @@ import { SharedModule } from './shared/shared.module';
 export class AppModule {}
 ```
 
-In contrast, when import the same module in our `LazyModule` we will not call the `forRoot` method because we don't want to register the service again in a different level of the DI tree, so the declaration of the `LazyModule` doesn't change.
+Since we import the module with `forRoot` in our root application module, we **do not** need to import it in `LazyModule`. Reimporting core root modules defined with `forRoot` in lazily loaded modules can cause issues in your application.
```

Comment:
We need to explain the issues or point to a section where the issue is explained